### PR TITLE
Kapa 1.5.8 release notes

### PR DESCRIPTION
**Pull request by kelseiv**
Closes https://github.com/influxdata/docs-v2/issues/1910

Update Kapacitor 1.5.8 release notes.

### CODE REVIEW

File: content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md [hunk 1]
```diff
@@ -6,6 +6,27 @@ menu:
...
+- Remove support for darwin/386 builds (Go no longer supports), and add support for darwin/arm64 builds.
+
+## Features
+
+- Add InfluxDB `subscription-path` configuration option to allow Kapacitor to run behind a reverse proxy, thanks @aspring!
+- Add support for HTTP sources in `SideloadNode` configuration, thanks @jregovic!
+- Add support for correlate in the Alerta AlertNode, thanks @nermolaev!
+- Add `details` option to the OpsGenie v2 event handler; set to `true` to use the Kapacitor alert details as OpsGenie description text, thanks @JamesClonk!
+- Preallocate GroupIDs for increased performance by reducing allocations.
+- Send data to InfluxDB compressed as `gzip` by default.
```
Comment (kelseiv) [inline content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:23]: Do folks have the option to change this in the Kapacitor config and/or elsewhere? If so, do we support other content encoding?

File: content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md [hunk 2]
```diff
@@ -6,6 +6,27 @@ menu:
...
+
+## Breaking changes
+
+- Remove support for darwin/386 builds (Go no longer supports), and add support for darwin/arm64 builds.
+
+## Features
+
+- Add InfluxDB `subscription-path` configuration option to allow Kapacitor to run behind a reverse proxy, thanks @aspring!
+- Add support for HTTP sources in `SideloadNode` configuration, thanks @jregovic!
+- Add support for correlate in the Alerta AlertNode, thanks @nermolaev!
```
Comment (kelseiv) [inline content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:20]: Is correlate a new event handler option? configuration option? and/or option to include in task template scripts? Do we have an example to show how to use correlate?

File: content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md [hunk 3]
```diff
@@ -6,6 +6,28 @@ menu:
...
+- Add `details` option to the OpsGenie v2 event handler; set to `true` to use the Kapacitor alert details as OpsGenie description text, thanks @JamesClonk!
+- Send data to InfluxDB compressed as `gzip` by default.
+- Add `PrimaryProperty` and `SecondaryProperty` methods to BigPanda AlertNode.
```
Comment (kelseiv) [inline content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:23]: Should we say we that we added the BigPanda event handler and link to new doc [here](https://github.com/influxdata/docs-v2/pull/2032/files#diff-393c4d189c7674fa7049bb544cfdd7d983f437b7853092a8ec34dd078dc1f306)?

Not clear why 
I'd like to improve the distinction/relationship between "event handlers" and "AlertNodes"

Comment (docmerlin) [reply content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:23]: I'm not sure what you mean here.

File: content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md (hunk 2 above)
Comment (docmerlin) [reply content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:20]: Yes

Comment (docmerlin) [reply content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:20]: I do not have any examples, sorry, the bonito folks will know more.

File: content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md (hunk 1 above)
Comment (docmerlin) [reply content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:22]: They can turn it off by setting compression to "none" in the appropriate influxdb section.

File: content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md (hunk 3 above)
Comment (kelseiv) [reply content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:23]: @docmerlin, are the following statements true:

- We added a new BigPanda event handler. 
- _BigPanda AlertNode_ is different than _BigPanda event handler_. 
- We added the PrimaryProperty` and `SecondaryProperty`methods to the _BigPanda AlertNode_ (implies the BigPanda AlertNode already existed).
- In our [AlertNode](https://docs.influxdata.com/kapacitor/v1.5/nodes/alert_node/) doc, we can add the two new BigPanda methods to the Property methods table.

Comment (docmerlin) [reply content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:23]: `BigPanda AlertNode is different than BigPanda event handler.`
They are the same thing.

Comment (kelseiv) [reply content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:23]: Thank you, @docmerlin!
Our docs currently describe _AlertNodes_ and _event handlers_ as two separate things...what I meant by it'd be nice to improve the docs that describe this (non) relationship.

File: content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md (hunk 1 above)
Comment (kelseiv) [reply content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:24]: Thank you, @docmerlin. Is "the appropriate influxdb section" supposed to be in the [Kapacitor configuration](https://github.com/influxdata/kapacitor/blob/master/etc/kapacitor/kapacitor.conf) or elsewhere? Want to tell folks where they can update this setting.

Comment (docmerlin) [reply content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:24]: Yes in the kapacitor config were they set how to communicate with influx.

Comment (kelseiv) [reply content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md:24]: @docmerlin, I don't see gzip in the config...should it be there?

File: content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md (hunk 2 above)
Comment (kelseiv) [reply content/kapacitor/v1.5/about_the_project/releasenotes-changelog.md]: Thank you, @docmerlin, I don't see "correlate" as an option in the alerta section of the Kapacitor config. Should that be there?

**Pull request merged**